evento/app/controllers/users_controller.rb

```diff
@@ -63,7 +64,11 @@ def set_user
   end
 
   # Only allow a trusted parameter "white list" through.
-  def user_params
-    params.require(:user).permit(:name, :email, :password)
+  def create_params
```

Comment:
I didn't get it to work with user_params and didn't understand how it could work. If I understood correctly, the following code: 

`params.require(:user).permit(:name, :email, :password)`

requires that there is a params.user object that can have name, email and password (so params.user.name, params.user.password, params.user.email), but was that really the point?

If I missed something, then feel free to fix. But my changes didn't break any tests (probably since none of our tests prior to my commits tested put/update and we still dont have any tests for creating new users) and seemed more logical to me.